src/frontend/src/lib/providers/coingecko/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './CoinGeckoClient';
```

Comment:
please avoid rexporting an index module, this is bad for chunking as it will import everything. prefer explicit import and remove this `index.ts` for consistency reason as well